src/SourceBuild/content/build.proj

```diff
@@ -57,6 +58,82 @@
     <MSBuild Projects="$(RepoProjectsDir)$(RootRepo).proj" Targets="WritePrebuiltUsageData;ReportPrebuiltUsage" />
   </Target>
 
+  <Target Name="DiscoverSymbolsTarballs"
+          AfterTargets="Build">
+    <ItemGroup>
+      <SymbolsTarball Include="$(OutputPath)Symbols.*.tar.gz" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="ExtractSymbolsTarballs"
```

Comment:
This should have a `DependsOnTargets` for `DiscoverSymbolsTarballs`.

Reply:
I don't think it's necessary due to target order in `DependsOnTargets` attribute of `RepackageSymbols` target.

But, it won't hurt to have it here as well.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17454/commits/abc74a8046a368256c35911490a9db91212aaac0